File: packages/javascript/lib/date/today.ts
export function today(separator = '-', yearFirst = true): string {
	const date = new Date();
	const month = date.getMonth() + 1;
	const day = date.getDate();

	const dateArr = [`${month < 10 ? '0' : ''}${month}`, `${day < 10 ? '0' : ''}${day}`];

	if (yearFirst) {
		dateArr.unshift(date.getFullYear().toString());
	} else {
		dateArr.push(date.getFullYear().toString());
	}

	return dateArr.join(separator);
}

File: packages/python/qsu/date/today.py
from datetime import datetime


def today(separator: str = '-', yearFirst: bool = True) -> str:
	date = datetime.now()
	month = date.month
	day = date.day

	date_arr = [
		f"{'0' if month < 10 else ''}{month}",
		f"{'0' if day < 10 else ''}{day}",
	]

	if yearFirst:
		date_arr.insert(0, str(date.year))
	else:
		date_arr.append(str(date.year))

	return separator.join(date_arr)
